pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -229,15 +230,90 @@ func getQuotaRequester(orgId int64) *identity.StaticRequester {
 }
 
 func (dr *DashboardServiceImpl) GetProvisionedDashboardData(ctx context.Context, name string) ([]*dashboards.DashboardProvisioning, error) {
+	if dr.features.IsEnabledGlobally(featuremgmt.FlagKubernetesCliDashboards) {
+		orgs, err := dr.orgService.Search(ctx, &org.SearchOrgsQuery{})
+		if err != nil {
+			return nil, err
+		}
+
+		results := []*dashboards.DashboardProvisioning{}
+		for _, org := range orgs {
```

Comment:
NIT – we could probably use `errgroup.WithContext` to search concurrently across orgs.

Reply:
Good point - updated!